File: Lua/Original_naive/Original_naive.lua
function naiveMultiplier(val){ 
		var result, i; 
		var multiplier=val;
		//Creating Array object which stores results 
		var result_table = new Array(); 
		for (i=0; i<10; i++){ 
			result=multiplier*(i+1); 
			result_table.push(result); 
		} 
		return result_table; 
} 

var res; 
test = naiveMultiplier(7); 
console.log(test.toString());

File: Nodejs/Original_naive/Original_naive.js
function naiveMultiplier(val)
    -- Creating table object which stores results
    result_table={}
    for i = 0, 9, 1 do
      temp=val*(i+1)
      table.insert(result_table,temp) 
    end
    return result_table
end

local function main()
    res=naiveMultiplier(7)
    print(table.concat(res,","))
end
main()

